### DISCUSSION TITLE
Figma Design System

### DISCUSSION THREAD

**Discussion by ImBIOS** [saadeghi/daisyui#1027] [Ideas / feature requests]
How if we create Figma Design system for DiasyUI for easy prototyping of the UI?

**Comment by saadeghi** [comment]
I have no plans to create a Figma file myself.  
But any effort from the community is appreciated.  

I haven't used a design tool to create UI since Tailwind came out. It's much faster and easier to design with utility classes. I only use Figma to work with SVGs, icons and images.  

There's also [this unfinished Figma file](https://www.figma.com/community/file/1052493195038031908). It might be useful.

---

Update: The offical daisyUI Figma file is in development and will be ready soon

**Reply by abheist** [reply]
Great if some designer picks it up, though hard to find someone on GitHub 😅

**Reply by fabriciop-carvalho** [reply]
Hey, guys! I'm currently doing this!

I'm a Designer and appreciate the Daisy UI community, and I know a lot of developers who would benefit from Daisy UI in Figma.

btw, I'm here looking to understand more about how Daisy UI works for developers lol

If you guys want to see more, or contribute with me, here is the link to my current work with Daisy UI in Figma

https://www.figma.com/community/file/1246877849405420060

**Comment by z3tsubouou** [comment]
I think daisyui needs Figma design system. Because v3 is now out and it is getting popular. Also now I am recommending daisyui to my team but daisyui doesn't have Figma design system so my team decided not to use daisyui.

@saadeghi

**Reply by saadeghi** [reply]
You might be able to use a HTML to Figma plugin maybe?  
daisyUI is made to speedup the development by skipping the design step. I'm not sure how putting back an already-designed element back to Figma would help. You can prototype in Figma, then use daisyUI + Tailwind CSS for development. The prototype doesn't need to be realistic and making it realistic is a waste of time in my opinion.

**Reply by abheist** [reply]
Agree on this part, It'll be great to have a Figma plugin and not mandatory.

Currently as per my understanding, DaisyUI is mostly getting used by developers who are working on side-projects and all. If daisyUI needs to grow and to reach the teams who have design in their process, then it'll be hard to daisyUI to get adopted by these teams.

**Reply by saadeghi** [reply]
@abheist Maybe.  
If anyone is willing to contribute on a Figma file (or any other design asset) I would gladly welcome them but I can't do it myself. 
I have to work in daisyUI itself. There are bugs and needed improvements and I also have to work on my freelance projects as well. 
 daisyUI is still a free and non-profit project.

**Reply by fabriciop-carvalho** [reply]
As I said in [this last answer](https://github.com/saadeghi/daisyui/discussions/1027#discussioncomment-6137627), I'm a Designer and I started doing this in Figma.

Hope you like it!

https://www.figma.com/community/file/1246877849405420060

**Reply by saadeghi** [reply]
@fabriciop-carvalho that's great 👍  
Let me know when it's finished so I can add a link to website's footer 🙌

**Reply by Mircobrb** [reply]
@fabriciop-carvalho i've just seen the figma file and it looks cool👍 if you need help i can lend a hand building some of the components in figma. Let me know if i can contribute in any way 🤙

**Reply by NayamAmarshe** [reply]
Awesome! Thanks a lot @fabriciop-carvalho !

**Reply by Mircobrb** [reply]
If anyone is still interested I have made a copy of the file by @fabriciop-carvalho and released some components, here's the file: https://www.figma.com/community/file/1252306778489709049/DaisyUI---Components-(WIP)

I have also been in contact with the [Unlock design team](https://www.linkedin.com/company/unlock-technologies/) and they are eager to continue the work i've started an expand it, so stay tuned i guess 👍

**Reply by mlaksi** [reply]
I'm a UI/UX enthusiast and junior front-end dev, I'm willing to lend you a hand while building this Figma file, message me to plan how we could split the work! Best regards, Mladenka

**Reply by Mircobrb** [reply]
Hi @mlaksi , sorry for the late reply, sure thing you can contact me on twitter https://twitter.com/MircoBrB or just send me your email 👍

**Comment by thebeafiles** [comment]
is there any up to date versions? i'm thinking of introducing it to my team but, honestly it not having an up to date design system makes it quite difficult.

**Reply by saadeghi** [reply]
We're working on the official Figma file. It will include everything, it will be updated regularly and it is expected to be released soon.

**Reply by lirc571** [reply]
@saadeghi thank you so much for the great work! Any ETA?

**Reply by rakeshmukundan** [reply]
Wonderful news! We are also eagerly waiting for this

**Reply by piotrzarzycki21** [reply]
> We're working on the official Figma file. It will include everything, it will be updated regularly and it is expected to be released soon.

Hi @saadeghi Is there any progress on this ? In your first comments you were providing WIP link to current progress, can you share one again ?

**Reply by saadeghi** [reply]
@piotrzarzycki21  
Hey,  
The WIP link is not our work. it was a community project. It's the same link as above (last update 2023)

The Official daisyUI Figma Library is available here:  
https://daisyui.com/store/351127/
